battleships/arbitrum-goerli.yaml

```
@@ -0,0 +1,3 @@
+dapp:
+  image: docker.io/productiv3/cartesian-battleships:battleships-0.6.0-machine
```

We need the `-server` prefixed image to run the backend. Make sure to publish it to the Docker Registry.

```suggestion
  image: docker.io/productiv3/cartesian-battleships:battleships-0.6.0-server
```